docs/library/theodore.md

```diff
@@ -103,17 +107,17 @@ The Theodore core supports the following device type(s) in the controls menu, bo
 
 ## Other devices
 
-- Light pen - The TO8 light pen inputs are mapped to the mouse
+- Light pen - The TO8 light pen inputs are mapped to the mouse.
 
 ## Joypad
 
-| RetroPad Inputs                        | User 1 input descriptors    |
-|----------------------------------------|-----------------------------|
-| ![](../image/retropad/retro_a.png)     | "Fire" button               |
-| ![](../image/retropad/retro_b.png)     | Start button                |
-| ![](../image/retropad/retro_x.png)     | Virtual keyboard: go up     |
-| ![](../image/retropad/retro_y.png)     | Virtual keyboard: go down   |
-| ![](../image/retropad/retro_start.png) | Virtual keyboard: keystroke |
+| RetroPad Inputs                        | User 1 input descriptors       |
+|----------------------------------------|--------------------------------|
+| ![](../image/retropad/retro_a.png)     | "Fire" button                  |
+| ![](../image/retropad/retro_b.png)     | Start screen only: Run content |
```

Comment:
B button is just really a shortcut for the B letter on the keyboard. If you run the emulator without a disk loaded, start one of the BASIC interpreters and press B on the pad you will just see lots of B letters on the screen. B allows to start "most" of the games (I would say 95%) but not all. For example, the program on a cartridge is generally loaded with the A key (the corresponding entry appears on the main screen only when a cartridge is loaded). Other programs may require the 2nd BASIC to start (C key).